test/swmr.jl

```diff
@@ -114,4 +116,9 @@ end
 end
 
 rm(fname) # cleanup file created by swmr tests
+
+if nprocs() > 1
+    rmprocs(procs)
```

Comment:
This solves the stall on Julia 1.8 nightly. See https://github.com/JuliaIO/HDF5.jl/pull/881

Reply:
I propose  pushing this is a separate commit.

Reply:
You want the test/swmr.jl in separate branch/PR or do you really mean a distinct commit?

Reply:
Sorry I mean a different PR, if it wasn't clear.

Reply:
I put it into #882